src/core/Core.cpp

```diff
@@ -5,5 +5,53 @@ namespace bakcyl::core {
 Core::Core()
 {
 }
-
+    std::uint32_t Core::getProductCurrentQuantity(const std::uint64_t &id) const

```

Comment:
Dotyczy  to większości z tych metod.

1. Po co przekazujesz id, skoro go nie używasz?
2. Typy proste, takie jak np. int'y nie muszą być przekazywane przez referencję. https://stackoverflow.com/questions/14013139/is-it-counter-productive-to-pass-primitive-types-by-reference